daemon/call_interfaces.c

```diff
@@ -36,7 +36,7 @@ static pcre_extra *streams_ree;
 
 int trust_address_def;
 int dtls_passive_def;
-
+int pad_crypto_def;
```

Comment:
I don't see a way to set this to anything other than the default 0? I think this can be omitted unless you also want to add a global config option (in which case it should go into rtpe_config).

Reply:
removed the unnecessary pad_crypto_def variable

Reply:
Will it be possible to merge into mr6.5 branch?